docs/source/getting-started/imi-config-file.rst

```diff
@@ -247,11 +248,18 @@ Note: some python scripts are also deployed using slurm and default to using the
      - Number of cores to allocate to each jacobian simulation (run in parallel).
    * - ``JacobianMemory``
      - Amount of memory to allocate to each jacobian simulation (in MB).
+   * - ``RequestedTime``
+     - Max amount of time to allocate to each sbatch job (eg. "0-6:00")
    * - ``SchedulerPartition``
      - Name of the partition(s) you would like all slurm jobs to run on (eg. "debug,huce_intel,seas_compute,etc").
    * - ``MaxSimultaneousRuns``
      - The maximum number of jacobian simulations to run simultaneously. The default is -1 (no limit) which will submit all jacobian simulations at once. If the value is greater than zero, the sbatch array statement will be modified to include the "%" separator and will limit the number of simultaneously running tasks from the job array to the specifed value.
- 
+   * - ``NumJacobianRuns``
```

Comment:
Add documentation for `CombineJacobianRuns` too?

Reply:
CombineJacobianRuns has been removed during cleanup. I fixes the description of this PR above to reflect that.